.threeport/sdk-config.yaml

```diff
@@ -0,0 +1,211 @@
+APIObjects:
+  secret:
+    - Name: SecretDefinition
+      Versions:
+        - v0
+      Reconcilable: true
+      AllowCustomMiddleware: true
+      DisableNotificationPersistence: true
+    - Name: SecretInstance
+      Versions:
+        - v0
+      Reconcilable: true
+  actuator:
+    - Name: Profile
+      Versions:
+        - v0
+    - Name: Tier
+      Versions:
+        - v0
+  aws:
+    - Name: AwsAccount
+      Versions:
+        - v0
+      Tptctl: true
+    - Name: AwsEksKubernetesRuntimeDefinition
+      Versions:
+        - v0
+      Tptctl: true
+    - Name: AwsEksKubernetesRuntimeInstance
+      Versions:
+        - v0
+      Reconcilable: true
+      Tptctl: true
+    - Name: AwsRelationalDatabaseDefinition
+      Versions:
+        - v0
+      Tptctl: true
+    - Name: AwsRelationalDatabaseInstance
+      Versions:
+        - v0
+      Reconcilable: true
+      Tptctl: true
+    - Name: AwsObjectStorageBucketDefinition
+      Versions:
+        - v0
+      Tptctl: true
+    - Name: AwsObjectStorageBucketInstance
+      Versions:
+        - v0
+      Reconcilable: true
+      Tptctl: true
+  control_plane:
+    - Name: ControlPlaneDefinition
+      Versions:
+        - v0
+      Reconcilable: true
+      Tptctl: true
+    - Name: ControlPlaneInstance
+      Versions:
+        - v0
+      Reconcilable: true
+      LoadAssociationsFromDatabase: true
+      Tptctl: true
+  control_plane_components:
+    - Name: ControlPlaneComponent
+      Versions:
+        - v0
+      RouteExclude: true
+      DisableApiModel: true
+  class:
+    - Name: Definition
+      Versions:
+        - v0
+      RouteExclude: true
+      DisableApiModel: true
+    - Name: Instance
+      Versions:
+        - v0
+      RouteExclude: true
+      DisableApiModel: true
+  gateway:
+    - Name: GatewayDefinition
+      Versions:
+        - v0
+      Reconcilable: true
+      Tptctl: true
+    - Name: GatewayInstance
+      Versions:
+        - v0
+      Reconcilable: true
+      Tptctl: true
+    - Name: GatewayHttpPort
+      Versions:
+        - v0
+    - Name: GatewayTcpPort
+      Versions:
+        - v0
+    - Name: DomainNameDefinition
+      Versions:
+        - v0
+    - Name: DomainNameInstance
+      Versions:
+        - v0
+      Reconcilable: true
+      Tptctl: true
+  helm_workload:
+    - Name: HelmWorkloadDefinition
+      Versions:
+        - v0
+      Reconcilable: true
+      TptctlConfigPath: true
+    - Name: HelmWorkloadInstance
+      Versions:
+        - v0
+      Reconcilable: true
+      TptctlConfigPath: true
```

Comment:
Merge with other tptctl to single struct:

type Tptctl struct {
	enabled  bool `yaml:",inline"`
	configPath bool
}